accumulo/src/main/java/org/vertexium/accumulo/AccumuloGraph.java

```diff
@@ -1628,19 +1737,35 @@ public Connector getConnector() {
         }
     }
 
+    public Iterable<Range> listHistoryVerticesTableSplits() {
+        try {
```

Comment:
Consider making a private method `listTableSplits(String tableName)`? All of the various list*TableSplits methods would then be one-liner calls to that.